src/components/gcds-checkbox/gcds-checkbox.tsx

```diff
@@ -73,21 +66,130 @@ export class GcdsCheckbox {
       this.errorMessage = "";
     } else if (!this.hasError && this.errorMessage) {
       this.hasError = true;
+    } else if (this.errorMessage == "") {
+      this.hasError = false;
     }
   }
```

Comment:
Can you turn this whole function into a util function since it is used in the exact same way by:
- gcds-file-uploader
- gcds-fieldset
- gcds-select

Reply:
Can you do the same with the `validateDisabledCheckbox` function on L43? It is also the exact same function used by:
- gcds-checkbox
- gcds-file-uploader
- gcds-fieldset
- gcds-select